css/style.css

```
@@ -16,4 +15,98 @@ body {
  * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
  */
 
+nav {
+  padding: 20px 40px 3px 60px;
+  background-color: rgb(247, 244, 241);
+  display: flex;
+  flex-direction: row;
+  flex-wrap: nowrap;
+  justify-content: space-between;
+}
+.nav_link > img {
```

```suggestion
.nav-link > img {
```
CSS class names are usually in [kebab-case](https://winnercrespo.com/naming-conventions/#kebab-case), so we can follow this convention too